resourcemgr/resourcemgr.c

```diff
@@ -1932,6 +1933,12 @@ TSS2_RC ResourceMgrReceiveTpmResponse(
                                 foundEntryPtr->hierarchy, cmdConnectionId );
                         if( rval != TSS2_RC_SUCCESS )
                             goto returnFromResourceMgrReceiveTpmResponse;
+
+                        // Find entry for the new persistent object and flag it as persistent
+                        rval = FindEntry( entryList, RMFIND_REAL_HANDLE, persistentHandle, &foundEntryPtr );
+                        if( rval != TSS2_RC_SUCCESS )
+                            goto returnFromResourceMgrReceiveTpmResponse;
+                        foundEntryPtr->status.persistent = 1;
```

Comment:
I agree that this code change or something like it is needed.  I do need to set the persistent bit here, since I use it elsewhere.  

But...what I'd rather do is just eliminate the persistent bit and always look at the handle type when trying to decide if a handle is persistent or not..